formsfeeder.client/src/main/java/formsfeeder/client/support/BuilderImpl.java

```diff
@@ -72,6 +73,15 @@ public BuilderImpl basicAuthentication(String username, String password) {
 		return this;
 	}
 
+	@Override
+	public BuilderImpl addQueryParam(String name, Supplier<Object> value) {
+		this.queryParams.put(name,value);
+		return this;
+	}
+
+	@Override
+	public Map getQueryParams() { return this.queryParams; }
+

```

Comment:
This gives me a warning - needs type information on the Map (i.e. `Map<String, List<Supplier<String>>>`).